builtin/providers/rancher/resource_rancher_environment.go

```diff
@@ -203,6 +261,39 @@ func setOrchestrationFields(orchestration string, data map[string]interface{}) {
 	data[orch] = true
 }
 
+func normalizeMembers(in []rancherClient.ProjectMember) (out []interface{}) {
+	for _, m := range in {
+		mm := map[string]string{
+			"external_id_type": m.ExternalIdType,
+			"external_id":      m.ExternalId,
+			"role":             m.Role,
+		}
+		out = append(out, mm)
+	}
+	return
+}
+
+func makeProjectMembers(in []interface{}) (out []interface{}) {
+	for _, m := range in {
```

Comment:
`out` needs to be initialized outside the loop, lest it is not available as a return value after the loop is finished.